components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -890,7 +890,8 @@ func (backend *ES2Backend) GetControlListItems(ctx context.Context, filters map[
 						controlSummaryTotals.Waived.Total = int32(totalWaivedControls.DocCount)
 					}
 					if controlBuckets, found := filteredControls.Aggregations.Terms("control"); found && len(controlBuckets.Buckets) > 0 {
-						for _, controlBucket := range controlBuckets.Buckets {
+						start, end := paginate(int(pageNumber), int(size), len(controlBuckets.Buckets))
+						for _, controlBucket := range controlBuckets.Buckets[start:end] {
```

Comment:
Please cross-check once here, a[start:end] will return elements from start to end-1 only. 
is that what you looking for? you may have to use a[start:]

please re check once.

Reply:
@Dmaddu a[start:end] will return elements from start to end-1 but it will work, because end is the capacity of that slice.